tests/python/multidevice/fusion_definition_wrapper.py

```diff
@@ -19,57 +19,47 @@ def make_key_from_dtensors(dtensors: Iterable[DTensor]) -> DTensorsKey:
 
 
 class FusionDefinitionWrapper:
-    def __init__(self, define_fusion: Callable[[nvfuser.FusionDefinition], None]):
+    def __init__(self, define_fusion: Callable):
         """Wraps a function that defines a fusion without `multidevice_schedule`."""
         # The complete FusionDefinition (w/ multidevice_scheduler) will have to
         # be created at "call" time according to the input DTensors.
         self._define_fusion = define_fusion
 
-        # In theory, a FusionDefinitionWrapper can own multiple
+        # TODO: In theory, a FusionDefinitionWrapper can own multiple
```

Comment:
I left updating the `FusionDefinitionWrapper` to cache based of fusion definition for future PR. It has the same behavior of caching on inputs.